README.md

```diff
@@ -375,10 +381,63 @@ make kill_all_port_forwardings
 
 ## Test `assisted-service` image
 
+### Initial Deployment with Custom Image
+
 ```bash
 make destroy run SERVICE=<image to test>
 ```
 
+### Updating a running `assisted-service` deployment
```

Comment:
I believe these instructions should be located in some file called `dev.md` or something similar. This `README.md` is too long already. Maybe we can add a pre-issue to this issue about splitting the README.md file to multiple files inside a `docs` directory ? @AlonaKaplan WDYT ?

Reply:
We can have a ref from here to another file to avoid this README to be even longer.